### ISSUE TITLE
Have POST to /api/admin/users return JSON response with new user ID

### ISSUE THREAD

**Issue by hartfordfive** [grafana/grafana#3237]
It would be convenient if this method returned the user ID upon successful creation.  Example:

```javascript
{"message": "User created", "id": 10}
```

This would be useful in scenarios such as those where you create users and immediately want to give them a different permission or Grafana admin privilege. 

I know the creation occurs is in `pkg/services/sqlstore/user.go` although I haven't analyzed it in depth enough to see how complex this would be to implement.

**Comment by utkarshcmu** [comment]
I am trying to replicate the issue you are facing here. I am trying the following URL to create a user:

```
curl -X POST -d '{"name":"micku","password":"mickuticku","login":"micku", "email":"micku@gmaik.com"}' http://admin:admin@localhost:3000/api/admin/users
```

But the response I am getting back is:
```
[{"fieldNames":["Password"],"classification":"RequiredError","message":"Required"}]
```

Am I missing anything? Can you please post your Curl request please?

**Comment by woodsaj** [comment]
@utkarshcmu you need to include a "content-type: application/json" header when posting data.

**Comment by utkarshcmu** [comment]
@woodsaj Thanks. Works for me now. :)

**Comment by utkarshcmu** [comment]
PR #3254 submitted, please review/merge.

**Comment by utkarshcmu** [comment]
This issue can be closed now as #3254 is merged.
